src/ShopListFilterBundle/objects/FilterApi.php

```diff
@@ -198,7 +198,9 @@ public function getCacheTrigger()
     }
 
     /**
-     * @return string
+     * @return string|null
+     *
+     * @FIXME The result of `->getActivePage()` can be null, which means the property fetch on it may be null as well (currently yielding a warning, but may be a fatal error in future PHP Version?)
```

Comment:
This is a bug in the making: It is very possible, that future PHP Versions will handle feching properties on `null` as a `null` return with a warning but as a `FatalError`